CHANGELOG.md

```diff
@@ -1,8 +1,51 @@
-Changelog
-=========
+# Changelog
 
-## Unreleased
-[Full Changelog](https://github.com/pcfens/puppet-ca_cert/compare/v2.5.0...HEAD)
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
+
+## [v3.0.0](https://github.com/voxpupuli/puppet-ca_cert/tree/v3.0.0) (2024-06-06)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-ca_cert/compare/v2.5.0...v3.0.0)
+
+**Breaking changes:**
+
+- Support distrusting ca's on Debian; Changed ca\_cert::ca parameter names and meaning [\#107](https://github.com/voxpupuli/puppet-ca_cert/pull/107) ([h-haaks](https://github.com/h-haaks))
+- Remove defaults for AIX and Solaris as we can't verify/maintain these [\#100](https://github.com/voxpupuli/puppet-ca_cert/pull/100) ([h-haaks](https://github.com/h-haaks))
+- Move and use params only in ca\_certs class [\#99](https://github.com/voxpupuli/puppet-ca_cert/pull/99) ([h-haaks](https://github.com/h-haaks))
+- Drop EoL Suse support [\#97](https://github.com/voxpupuli/puppet-ca_cert/pull/97) ([h-haaks](https://github.com/h-haaks))
+- Drop support for EoL Ubuntu versions [\#88](https://github.com/voxpupuli/puppet-ca_cert/pull/88) ([bastelfreak](https://github.com/bastelfreak))
+- Drop EoL EL support [\#87](https://github.com/voxpupuli/puppet-ca_cert/pull/87) ([bastelfreak](https://github.com/bastelfreak))
+- Drop Puppet 4/5/6 support [\#86](https://github.com/voxpupuli/puppet-ca_cert/pull/86) ([bastelfreak](https://github.com/bastelfreak))
+
+**Implemented enhancements:**
+
+- Support proxy when downloading remote certificates [\#108](https://github.com/voxpupuli/puppet-ca_cert/pull/108) ([h-haaks](https://github.com/h-haaks))
+- Add SLES and OpenSUSE 15 support [\#104](https://github.com/voxpupuli/puppet-ca_cert/pull/104) ([h-haaks](https://github.com/h-haaks))
+- Add supported OSes [\#94](https://github.com/voxpupuli/puppet-ca_cert/pull/94) ([h-haaks](https://github.com/h-haaks))
+- Add RedHat 9, CentOS, Rocky and AlmaLinux support [\#93](https://github.com/voxpupuli/puppet-ca_cert/pull/93) ([h-haaks](https://github.com/h-haaks))
+- enable acceptance tests [\#89](https://github.com/voxpupuli/puppet-ca_cert/pull/89) ([bastelfreak](https://github.com/bastelfreak))
+
+**Closed issues:**
+
+- Support distrusting ca's on Debian family [\#96](https://github.com/voxpupuli/puppet-ca_cert/issues/96)
+- Allow `Deferred` for the ca\_text parameter [\#90](https://github.com/voxpupuli/puppet-ca_cert/issues/90)
+- Add proxy support to ca\_cert::ca \(remote\_file\) [\#46](https://github.com/voxpupuli/puppet-ca_cert/issues/46)
```

Comment:
should this be skipped as well? Already in Implemented enhancements

Reply:
I'm no expert on this. But I got the impression that each change should only have one change log entry, so I'd say yes

Reply:
Removed :)